lxd/device/device_utils_disk.go

```diff
@@ -452,6 +453,7 @@ func DiskVMVirtiofsdStart(kernelVersion version.DottedVersion, inst instance.Ins
 		"--shared-dir", sharePath,
 		"--cache", "auto", // "never" and "metadata" modes do not allow execution at this time.
 		"--allow-direct-io",
+		"--thread-pool-size", strconv.FormatUint(uint64(threadPoolSize), 10),
```

Comment:
This one is supported by Jammy with the `=` in between, dunno if relevant as arg parsing is so unpredictable.